style.css

```diff
@@ -1 +1,217 @@
+
+body{
+    background-color: rgb(255, 255, 255);
+    font-family: Arial !important;
+}
+
+/* Header / Upper part */
+
+header{
+    width:100%;
+    display: flex;
+    justify-content: space-between;
+    background-color: rgb(255, 212, 229);
+    color: rgb(134, 134, 134);
+    margin-bottom: 20px;
+}
+
+.bakery-icon{
+    width: 100%;
+    display: flex;
+    justify-content: space-around;
+}
+
+.bakery-icon p{
+    font-size: calc(15px + 1vw);
+}
+
+.cupcake-icon img{
+    width:125px;
+    margin-top: 10px;
+
+}
+
+.welcome-msg{
+    flex-wrap: nowrap;
+    margin-top: 90px;
+    width: 100%;
+    float: left;
+    justify-content: right;
+    text-align: end;
+    font-weight: bold;
+}
+
+/* Navigation bar */
+
+#links{
+    display:none;
+}
+
+.hamburger-icon{
+    margin-top:16px;
+    left: -10px;
+    width: 80%;
+    
+}
+.menu-nav {
+    overflow: hidden;
+    background-color:rgb(207, 144, 155);
+    position: absolute;
+    display:none;
+    min-width: 100vw;
+    top:calc(65px + 3vw);
+}
+
+
+.menu-nav li{
+      list-style-type: none;
+
+}
+  
+.menu-nav a {
+    color: rgb(85, 85, 85);
+    padding: 25px 25px;
+    text-decoration: none;
+    font-size: 17px;
+    display: block;
+}
+
+/* Main */
+
+.welcome-image{
+    width: 100%;
+    height: 100%;
+    flex: auto;
+    margin-bottom: 25px;
+}
+
+.welcome{
+    width: 100;
+    text-align: justify;
+    text-justify: inter-word;
+    color: rgb(150, 150, 150);
+    
+}
+
+.products img{
+    width: 49%;
+    display: inline;
+    flex-direction: row;
+    justify-content: space-between;
+}
+
+/* Footer */
+
+footer{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin-top: 20px;
+    background-color: rgb(255, 212, 229);
+    color: rgb(255, 255, 255);
+    font-weight: bold;
+
+
+}
+
+.media-icons img{
+    width: 25px;
+}
+
+/* Media Query */
+@media(min-width:900px){
+
+    .bakery-icon{
+        justify-content: left;
+        
+    }
+
+    .welcome-msg{
+        text-align: right;
+        width: 155%;
+    }
+
+    .hamburger-icon{
+        display:none;
+    }
+
+    header{
+        flex-direction: column;
+    }
+
+    nav{
+        width: 100%;
+    }
+
+    .mobile-nav{
+        width: 100%;
+    }
+
+    /* Menu Options */
+
+    .menu-nav{
+        width: 100%;
+        display: block;
+        display: flex;
+        background-color: white;
+        margin-top: 70px;
+        
+
+    }
+    #links{
+        display: flex;
+    }
+
+    .menu-nav ul{
+        display:flex;
+        justify-content: flex-end;
+        width: 100%;
+        margin-right: 2rem;
+   }
+
+    .menu-nav ul li a:hover{
+       color:white;
+       background-color: pink;
+       
+    }
+
+    .menu-nav ul li a{
+      font-size: 15px;
+      background-color: rgb(134, 134, 134);
+      margin:0 5px;
+      
+   } 
+
+     /* Main */
+
+    .welcome{
+        display: flex;
+        justify-content: space-between;
+        margin: 2rem 2 rem ;
```

Comment:
This rule is not being applied because there is an extra space between `2` and `rem` 🙁 
You can inspect your CSS in the browser's dev tools to see what it actually being applied.